src/App.tsx

```diff
@@ -1,110 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import { DropDown } from './components/DropDown/DropDown';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [page, setPage] = useState(1);
+  const [elements, setElements] = useState<string[]>([]);
+
   return (
     <div className="container">
       <h1>Items with Pagination</h1>
 
       <p className="lead" data-cy="info">
-        Page 1 (items 1 - 5 of 42)
+        {`Page ${page} (items ${elements[0] && elements[0].split(' ')[1]} - ${elements[elements.length - 1] && elements[elements.length - 1].split(' ')[1]} of ${items.length})`}
```

Comment:
This line of code may cause an error if `elements[0]` or `elements[elements.length - 1]` is undefined, as you're trying to call `split` on it. You should add a check to ensure that the element exists before calling methods on it.